### PR TITLE
Release 1.4.6

### PR DESCRIPTION
**Pull request by ni-jfitzger**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- ~[ ] I've added tests applicable for this pull request~

### What does this Pull Request accomplish?
Release 1.4.6. See CHANGELOG.md for list of changes.

### List issues fixed by this Pull Request below, if any.
None

### What testing has been done?
PR Checks

### CODE REVIEW

**Comment by ni-jfitzger** [comment]
This should not be merged until:
* https://github.com/ni/nimi-python/pull/2008 has been merged
* The release has been uploaded to PyPI

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-* [Unreleased](#unreleased)
+* [1.4.6](#146---2023-09-12)
```
Comment (ni-jfitzger) [inline CHANGELOG.md:3]: I'm just guessing on the data, here. I'll change it, as needed.

**Review by marcoskirsch — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ni-jfitzger) [reply CHANGELOG.md]: Updated the date to today. The historical significance of the date is unfortunate, but not a good enough reason to make clients wait an extra day for the goods.

**Comment by ni-jfitzger** [comment]
No significant changes in the since the initial commit. No need to wait for testing to complete again.

**Pull request merged**